src/app/work-item/work-item.component.html

```diff
@@ -80,197 +80,8 @@ <h5 class="col-md-12 col-sm-12 col-xs-12">
-                    <ul class="dropdown-menu dropdown-menu-right">
-                      <li><a>Edit</a></li>
-                      <li><a>Start</a></li>
-                    </ul>
-                  </div>
-                </div>
-              </li>
-            </ul>
-          </div>
-          <div class="iterations">
-            <div class="iteration-header pointer"
-                 (click)="isCollapsedPastIteration = !isCollapsedPastIteration">
-              <div class="iteration-secondary-menu">
-                <h4><i class="fa fa-icon"
-                   [ngClass]="{'fa-angle-right': isCollapsedPastIteration,
-                               'fa-angle-down': !isCollapsedPastIteration}"></i>
-                Past Iteration</h4>
-              </div>
-            </div>
-            <ul class="not-current-iteration"
-                [collapse]="isCollapsedPastIteration">
-              <li class="iteration-subheader">
-                <div>
-                  <strong>iteration 3</strong>
-                </div>
-                <div>
-                  <span class="badge">10</span>
-                  <div class="dropdown dropdown-kebab-pf dib" dropdown>
-                    <button class="btn btn-link dropdown-toggle"
-                            type="button"
-                            aria-haspopup="true"
-                            aria-expanded="true"
-                            dropdown-open>
-                      <span class="fa fa-ellipsis-v"></span>
-                    </button>
-                    <ul class="dropdown-menu dropdown-menu-right">
-                      <li><a>Edit</a></li>
-                      <li><a>Start</a></li>
-                    </ul>
-                  </div>
-                </div>
-              </li>
-              <li class="iteration-subheader">
-                <div>
-                  <strong>iteration 5</strong>
-                </div>
-                <div>
-                  <span class="badge">10</span>
-                  <div class="dropdown dropdown-kebab-pf dib" dropdown>
-                    <button class="btn btn-link dropdown-toggle"
-                            type="button"
-                            aria-haspopup="true"
-                            aria-expanded="true"
-                            dropdown-open>
-                      <span class="fa fa-ellipsis-v"></span>
-                    </button>
-                    <ul class="dropdown-menu dropdown-menu-right">
-                      <li><a>Edit</a></li>
-                      <li><a>Start</a></li>
-                    </ul>
-                  </div>
-                </div>
-              </li>
-            </ul>
-          </div>
-        </div>
-        <div *ngIf="loggedIn" class="create-iteration-bar pointer">
-          <span><i class="pficon-add-circle-o fa-icon"></i>
-          <span class="create-iteration-label">Create Iteration</span></span>
-        </div>
+      <iteration></iteration>
```

Comment:
Component should be named with a prefix. So far we have been using **alm** but since it's not **alm** anymore, let's use **fabric8-planner**. The name should be **fabric8-planner-iteration**.
I am not sure if it supports number or not, please try and check.